pages.es/common/kill.md

```diff
@@ -0,0 +1,9 @@
+# kill
```

Comment:
could you please add all the examples from here? https://github.com/tldr-pages/tldr/blob/main/pages/common/kill.md

Reply:
done